gammapy/catalog/fermi.py

```diff
@@ -556,22 +569,177 @@ class SourceCatalogObject3FHL(SourceCatalogObject):
+
+        return ss
+
+    def _info_position(self):
+        """Print position info."""
+        d = self.data
+        ss = '\n*** Position info ***\n\n'
+        ss += '{:<20s} : {:.3f}\n'.format('RA', d['RAJ2000'])
+        ss += '{:<20s} : {:.3f}\n'.format('DEC', d['DEJ2000'])
+        ss += '{:<20s} : {:.3f}\n'.format('GLON', d['GLON'])
+        ss += '{:<20s} : {:.3f}\n'.format('GLAT', d['GLAT'])
+
+        # TODO: All sources are non-elliptical; just give one number for radius?
         ss += '\n'
-        ss += 'Detection significance : {}\n'.format(d['Signif_Avg'])
+        ss += '{:<20s} : {:.4f}\n'.format('Semimajor (95%)', d['Conf_95_SemiMajor'])
+        ss += '{:<20s} : {:.4f}\n'.format('Semiminor (95%)', d['Conf_95_SemiMinor'])
+        ss += '{:<20s} : {:.2f}\n'.format('Position angle (95%)', d['Conf_95_PosAng'])
+        ss += '{:<20s} : {:.0f}\n'.format('ROI number', d['ROI_num'])
+
+        return ss;
+
+    def _info_spectral_fit(self):
+        """Print model data."""
+        d = self.data
+        ss = '\n*** Model info ***\n\n'
+
+        fmt = '{:<32s} : {:.3f}\n'
+        args = ('Significance (10 GeV - 2 TeV)', d['Signif_Avg'])
+        ss += fmt.format(*args)
+
+        ss += '{:<32s} : {:.1f}\n\n'.format('Npred', d['Npred'])
+
+
+        ss += '{:<32s} : {}\n'.format('Spectrum type', d['SpectrumType'])
+
+        ss += '{:<32s} : {:.1f}\n'.format('Significance curvature', d['Signif_Curve'])
+
+        fmt = '{:<32s} : {:.3f} +- {:.3f}\n'
+        args = ('Spectral index', d['Spectral_Index'], d['Unc_Spectral_Index'])
+        ss += fmt.format(*args)
+
+        spec_type = d['SpectrumType'].strip()
+        if spec_type == 'LogParabola':
+            # ss += '{:<32s} : {:.3f} +- {:.3f}\n'.format('alpha', d['Spectral_Index'], d['Unc_Spectral_Index'])
+            ss += '{:<32s} : {:.3f} +- {:.3f}\n'.format('beta', d['beta'], d['Unc_beta'])
+
+        ss += '{:<32s} : {:.0f} {}\n'.format('Pivot energy', d['Pivot_Energy'].value, d['Pivot_Energy'].unit)
+
+        fmt = '{:<32s} : {:.3f} +- {:.3f}\n'
+        args = ('Power Law index', d['PowerLaw_Index'], d['Unc_PowerLaw_Index'], d['Unc_PowerLaw_Index'])
+        ss += fmt.format(*args)
+
+        unit = 'cm-2 GeV-1 s-1'
+        fmt = '{:<32s} : {:.3} +- {:.3} {}\n'
+        args = ('Flux Density at pivot energy', d['Flux_Density'].value, d['Unc_Flux_Density'].value, unit)
+        ss += fmt.format(*args)
+
+        unit = 'cm-2 s-1'
+        fmt = '{:<32s} : {:.3} +- {:.3} {}\n'
+        args = ('Integral flux (10 GeV - 1 TeV)', d['Flux'].value, d['Unc_Flux'].value, unit)
+        ss += fmt.format(*args)
+
+        unit = 'erg cm-2 s-1'
+        fmt = '{:<32s} : {:.3} +- {:.3} {}\n'
+        args = ('Energy flux (10 GeV - TeV)', d['Energy_Flux'].value, d['Unc_Energy_Flux'].value, unit)
+        ss += fmt.format(*args)
+
+        return ss
+
+    def _info_spectral_points(self):
+        """Print spectral points."""
+        ss = '\n*** Spectral points ***\n\n'
```

Comment:
Again, I'd suggest to remove the empty lines in these short methods that don't really have separate steps.
In addition, here it seems you dno't need the local variable `t`, because just putting the code on the next line works OK (ie. doesn't become too long in this case)